server/src/routes/clientRouter.js

```diff
@@ -3,7 +3,7 @@ import getClientIndexPath from "../config/getClientIndexPath.js";
 
 const router = new express.Router();
 
-const clientRoutes = ["/", "/services/:id", "/user-sessions/new", "/users/new"];
+const clientRoutes = ["/", "/services/:id", "/user-sessions/new", "/users/new", "/reviews"];
```

Comment:
Take a look at the routes defined in your React App.js - I don't see a route for `/reviews`, but rather some related to `services`